.env.example

```diff
@@ -1,3 +1,3 @@
-AD_CLIENT_ID: '<<Azure client ID>>'
-AD_TENANT_ID: '<<Azure tenant ID>>'
-MNESTIX_BACKEND_API_KEY: '<<YOUR_API_KEY>>'
\ No newline at end of file
+AD_SECRET_VALUE: '<<YOUR_SECRET>>'
+MNESTIX_BACKEND_API_KEY: '<<YOUR_API_KEY>>'
+NEXTAUTH_SECRET: '<<YOUR_SECRET>>'
```

Comment:
Is this even used anywhere?

Reply:
This is the template that other users can make their .env file